### ISSUE TITLE
Fix broken links in FAQ page

### ISSUE THREAD

**Issue by RohrerHope** [Whichost/frontend#167]
**What's to be fixed**:

![Screen Shot 2019-04-28 at 12 30 52](https://user-images.githubusercontent.com/39112974/56863691-91567100-69b1-11e9-9fd5-a0916784489d.jpg)

When you click on Fees page it takes back to the beginning of the FAQ and not to the Fees page!

**What's the expected behavior:**

When anyone clicks on a link that says Fees page, it should take them to the Fees page!

**What happens in reality (screenshot):**

It takes them back to the top of the FAQ page.

![Screen Shot 2019-04-28 at 12 32 55](https://user-images.githubusercontent.com/39112974/56863714-c662c380-69b1-11e9-9ca9-e4d67c691176.jpg)

**Priority (tag accordingly):** `urgent`
**Domain (tag accordingly):** `frontend`
**Tested on:** ?
**Branch**: `hotfix-fix-broken-links-in-FAQ-page`

**Work on it**

- Derive the branch from `master`. Prefix it with `hotfix-`.
- Tag with `ongoing` and move the card to `ongoing`.
- Fix the issue
- Commit your work
- Create a PR and ask for validation. Move the card to `staged`.
- Ask for validation.
- Close the issue

**Validate**

- Check on mobile and on desktop if the issue have been correctly fixed.
- If everything is correct, move the card to accepted column.
- If the issue isn't fixed, re-open the comment clearly what the problem is and what is the desired result. Move the card to `have issues` column.

Please follow https://github.com/Whichost/frontend/wiki/Git-workflow

**Comment by lcoenen** [comment]
This is a duplicate of #168

**Comment by lcoenen** [comment]
@RohrerHope The forms were on sharetribe, we need to wait for issue #136 to be fixed before we can handle that properly. 

In the meanwhile, this link will just redirect to the Contact Us form.
